diff --git a/src/libmain/unix/stack.cc b/src/libmain/unix/stack.cc
index 10f71c1dc..b079f5d70 100644
--- a/src/libmain/unix/stack.cc
+++ b/src/libmain/unix/stack.cc
@@ -8,6 +8,8 @@
 #include <unistd.h>
 #include <signal.h>
 
+#include <iostream>
+
 namespace nix {
 
 
@@ -30,7 +32,8 @@ static void sigsegvHandler(int signo, siginfo_t * info, void * ctx)
         ptrdiff_t diff = (char *) info->si_addr - sp;
         if (diff < 0) diff = -diff;
         if (diff < 4096) {
-            nix::stackOverflowHandler(info, ctx);
+            std::cerr << "Looks like a stack overflow - not handling" << std::endl;
+            //nix::stackOverflowHandler(info, ctx);
         }
     }
 
